conclave-host/src/main/kotlin/com/r3/conclave/host/EnclaveHost.kt

```diff
@@ -1041,6 +1056,8 @@ class EnclaveHost private constructor(
             val results = ArrayList<ScanResult>()
             EnclaveMode.values().mapNotNullTo(results) { findNativeEnclave(enclaveClassName, it) }
             findMockEnclave(enclaveClassName)?.let { results += it }
+            EnclaveMode.values()
```

Comment:
Have a single loop which does both graalvm above and gramine here.

Reply:
Done